src/garage/envs/normalized_env.py

```diff
@@ -102,62 +200,45 @@ def _apply_normalize_reward(self, reward):
         self._update_reward_estimate(reward)
         return reward / (np.sqrt(self._reward_var) + 1e-8)
 
-    def reset(self, **kwargs):
-        """Reset environment.
-
-        Args:
-            **kwargs: Additional parameters for reset.
-
-        Returns:
-            tuple:
-                * observation (np.ndarray): The observation of the environment.
-                * reward (float): The reward acquired at this time step.
-                * done (boolean): Whether the environment was completed at this
-                    time step.
-                * infos (dict): Environment-dependent additional information.
-
-        """
-        ret = self.env.reset(**kwargs)
-        if self._normalize_obs:
-            return self._apply_normalize_obs(ret)
-        else:
-            return ret
-
-    def step(self, action):
-        """Feed environment with one step of action and get result.
+    def __getattr__(self, attr):
```

Comment:
does this preserve documentation and enumeration for the methods it passes through? e.g. does `help(obj)` and `dir(obj)` still show me everything there is to know?